tests/Unit/Evolution/DiscontinuousGalerkin/Limiters/Test_HwenoImpl.cpp

```diff
@@ -583,7 +591,10 @@ void test_constrained_fit_2d_vector(
     }();
 
     // Fit procedure has somewhat larger error scale than default
-    Approx local_approx = Approx::custom().epsilon(1e-9).scale(1.);
+    // Error further increases when using Gauss points
+    const double gauss_tol =
+        (quadrature == Spectral::Quadrature::Gauss ? 100. : 1.);
+    Approx local_approx = Approx::custom().epsilon(gauss_tol * 1e-10).scale(1.);
```

Comment:
To explain this change: this was the only test that failed on my system when I first generalized to Gauss points, so an earlier commit had bumped the tolerance from 1e-10 to 1e-9. Now that there's more of these tests and I don't want to loosen the GaussLobatto tolerances excessively, I'm going back to 1e-10 times the correction.